cli/src/test/resources/ide-projects/update/project/settings/eclipse/plugins/anyedit.properties

```diff
@@ -0,0 +1,3 @@
+plugin_url=https://raw.githubusercontent.com/iloveeclipse/plugins/latest/

```

Comment:
these plugins are not used by the tests as far as I can see so only add what you really need.

Reply:
done